src/i18n/base.js

```diff
@@ -196,7 +196,10 @@ wb.i18nDict = {
 	"geo-allyttl": "@geo-allyttl@",
 	"geo-tgllyr": "@geo-tgllyr@",
 	"geo-hdnlyr": "@geo-hdnlyr@",
+
+	/* geo-bmapurl to be deprecated - do not use */
 	"geo-bmapurl": "@geo-bmapurl@",
+	"geo-bmap-url": "@geo-bmap-url@",
```

Comment:
why is this changing, and deprecated how?

Reply:
@LaurentGoderre it looks it did affect compatibility because all the existing maps are now broken: http://wet-boew.github.io/wet-boew/demos/geomap/geomap-en.html

Either we need to add that key back or merge this PR. Thoughts?